src/main/java/uk/gov/companieshouse/acsp/manage/users/controller/AcspMembershipsController.java

```diff
@@ -52,7 +62,43 @@ public ResponseEntity<AcspMembershipsList> findMembershipsForUserAndAcsp(
       final String acspNumber,
       final Boolean includeRemoved,
       final RequestBodyLookup requestBody) {
-    return null;
+    Map<String, Object> logMap = new HashMap<>();
+    logMap.put(REQUEST_ID_KEY, requestId);
+    logMap.put(ACSP_NUMBER_KEY, acspNumber);
+    logMap.put("includeRemoved", includeRemoved);
+    logMap.put("userEmail", requestBody.getUserEmail());
+    LOG.info("Getting members for ACSP & User email", logMap);
+
+    if (Objects.isNull(requestBody.getUserEmail())) {
+      throw new BadRequestRuntimeException(PLEASE_CHECK_THE_REQUEST_AND_TRY_AGAIN);
```

Comment:
you might want to log the error so that we know what caused the bad request.

Reply:
good point, have included in https://github.com/companieshouse/acsp-manage-users-api/pull/36/commits/f729ee4e31d19e71df8061596c397b616453662c along with a rebase on main to solve conflicts